frontend/src/app/application-forms/tree-application-form/tree-permit-rules/tree-permit-rules.component.html

```diff
@@ -0,0 +1,9 @@
+<div>
```

Comment:
So the permit-rules only appear on the application form, not on the guidelines page.  It makes sense to put them together if we want to display both on the guidelines. Should we do that?

Reply:
Is there any reason to exclude it from the guidelines page? It seems like it would provide a more consistent user experience to show the same rules whereever they are displayed as well as minimize the component complexity. @MelissaBraxton @bryanmcfadden @JaneZC

Reply:
This seems to align with @JaneZC was saying in this card: https://github.com/nciinc/fs-permit-platform/issues/321